packages/cli/src/base.ts

```diff
@@ -206,6 +206,7 @@ export abstract class BaseCommand extends Command {
       : getGasCurrency(this.config.configDir)
 
     const setStableTokenGas = async (stable: StableToken) => {
+      // TODO(Arthur): Update implementation to match new `setFeeCurrency` method signature
       await kit.setFeeCurrency(stableTokenInfos[stable].contract)
```

Comment:
Good point, since this is the CLI, we can change the implementation without affecting the user experience.
I can't find the note about deprecating methods you mentioned 🤔 Could you send me a link to it? Maybe I'm missing something 👀

Reply:
Oh, I see, do you mean removing the `gasCurrency` flag in general?

https://github.com/celo-org/developer-tooling/blob/a1e8387ae16b3792c4e195cc7a518d665200e126/packages/cli/src/base.ts#L204-L207

I'm looking at where the flag is used across the CLI.

Reply:
oops i thought i was looking at contract kit when i wrote that

Reply:
Understood 👍 
At the moment, my thinking is to update the `gasCurrency` flag to accept arbitrary addresses (`0x...`), instead of symbols (`cUSD`) as is currently the case:

https://github.com/celo-org/developer-tooling/blob/a1e8387ae16b3792c4e195cc7a518d665200e126/packages/cli/src/commands/config/set.ts#L21-L22